From 2bf112fba95271759034439ec55965dd49b8062e Mon Sep 17 00:00:00 2001
From: Sammy He <r62914@freescale.com>
Date: Tue, 26 Aug 2008 16:51:05 +0800
Subject: [PATCH] ENGR00089144 MX51: Set VPU clock to 133MHZ for TO1.0 board

Set VPU clock to 133MHZ for TO1.0 according to IC suggestion

Signed-off-by: Sammy He <r62914@freescale.com>
---
 arch/arm/mach-mx51/clock.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-mx51/clock.c b/arch/arm/mach-mx51/clock.c
index 96ebdd2..324cd81 100644
--- a/arch/arm/mach-mx51/clock.c
+++ b/arch/arm/mach-mx51/clock.c
@@ -1685,7 +1685,7 @@ static int _clk_vpu_set_parent(struct clk *clk, struct clk *parent)
 static struct clk vpu_clk[] = {
 	{
 	 .name = "vpu_clk",
-	 .parent = &axi_a_clk,
+	 .parent = &ahb_clk,
 	 .set_parent = _clk_vpu_set_parent,
 	 .enable = _clk_enable,
 	 .enable_reg = MXC_CCM_CCGR5,
@@ -1695,7 +1695,7 @@ static struct clk vpu_clk[] = {
 	 },
 	{
 	 .name = "vpu_core_clk",
-	 .parent = &axi_a_clk,
+	 .parent = &ahb_clk,
 	 .set_parent = _clk_vpu_set_parent,
 	 .enable = _clk_enable,
 	 .enable_reg = MXC_CCM_CCGR5,
-- 
1.5.4.4

